moped-editor/src/views/projects/projectView/ProjectComponents/ComponentForm.js

```diff
@@ -12,18 +12,21 @@ import {
 } from "@mui/material";
 import DateFieldEditComponent from "../DateFieldEditComponent";
 import { CheckCircle } from "@mui/icons-material";
-import { ControlledAutocomplete } from "./utils/form";
 import { GET_COMPONENTS_FORM_OPTIONS } from "src/queries/components";
 import SignalComponentAutocomplete from "./SignalComponentAutocomplete";
 import {
   ComponentOptionWithIcon,
+  DEFAULT_COMPONENT_WORK_TYPE_OPTION,
   useComponentOptions,
   useSubcomponentOptions,
   usePhaseOptions,
   useSubphaseOptions,
   useComponentTagsOptions,
+  useWorkTypeOptions,
   useResetDependentFieldOnAutocompleteChange,
 } from "./utils/form";
+import ControlledAutocomplete from "./ControlledAutocomplete";
```

Comment:
it seemed like the right time to move this into it's own module since the form util has gotten large.